post_setup.py

```diff
@@ -0,0 +1,36 @@
+import os
+import urllib2
+
+
+def finddirectory():
+    try:
+        import natlinkstatus
+        status = natlinkstatus.NatlinkStatus()
+        userdir = status.CoreDirectory
+        if os.path.isdir(userdir):
+            directory = (os.path.dirname(userdir))
+            print("\nCaster: NatLink found.\n" "Defaulting to Dragon NaturallySpeaking Engine\n")
+            return directory
+    except Exception:
+        print("\nCaster: NatLink not found.\n" "Defaulting to Windows Speech Recognition Engine\n")
+        directory = os.path.join(os.environ['USERPROFILE'], "Desktop")
+        if os.path.isdir(directory):
+            return directory
+
+
+def download():
+    response = urllib2.urlopen('https://raw.githubusercontent.com/dictation-toolbox/caster/develop/_caster.py')
```

Comment:
Thanks I didn't catch that thinking that a PIP install would always have an Internet connection. Perhaps the error code would be caught by hip itself before post install. However that's no excuse for bad code! 

Originally I was trying to use `requests` but since it's not part of the standard library it causes issues with missing imports. So I simplified with `urllib2`.

Reply:
Personally, I don't think it's too much to ask that `requests` is already installed. It seems anecdotally that most projects doing URL-based transfers use it. I could see it being used internally by Caster as well in the future.

IIRC, `urllib2` was replaced/upgraded in Python 3, so using `requests` would also simplify things for the future in that regard.